code/modules/clothing/gloves/marine_gloves.dm

```diff
@@ -145,3 +145,64 @@
 	desc = "Gloves worn by commissars of the Imperial Army so that they do not soil their hands with the blood of their men."
 	icon_state = "red"
 	soft_armor = list(MELEE = 35, BULLET = 30, LASER = 30, ENERGY = 30, BOMB = 15, BIO = 10, FIRE = 20, ACID = 20)
+
+///see defibrillator.dm
+/obj/item/clothing/gloves/defibrillator
```

Comment:
bruh, I said put them in their own file to reduce bloat lol, not move it into a different bloated file.